index.rst

```diff
@@ -55,7 +55,7 @@ Python Examples
     :caption: NCL
 
     NCL Index <ncl/ncl_index/ncl_index>
-    NCL Applications <applications/ncl_applications.rst>
+    NCL Applications <ncl/ncl_entries/ncl_entries.rst>
```

Comment:
I'm a little confused by the "NCL Applications" reference here.  Is this like functions by category?

Reply:
Good point! I think it was originally called that because it was the NCL parts of the applications in the applications folder, but, uh, this PR does reorg that.

It's not even functions by category, currently it's just a list of all the NCL entries we've got (see the relocated ncl/ncl_entries/ncl_entries.rst`). I imagine we'll eventually make it look nicer and organize it, but for now, it's literally just where the NCL entries live. 

I don't think we're calling them NCL Applications anywhere else in -applications, and I agree it is confusing. We should totally rename it, but I don't have a great idea what we should call it (for now or in the long term).

What about:
- NCL Entries (what they're called internally, maybe not clear either)
- NCL to Python (not sure that indicates a category of pages exactly)
- NCL to Python Applications (don't love this either, sort of long)

Totally open to (and hoping for) suggestions here

Reply:
What about a structure like this?  With the "applications" bit implied.  Side note - at this point I think we're pulling from something a bit broader than just the index of "NCL Applications" scripts anyway so maybe not being so closely tied to that is fine and perhaps less confusing for those w/out the pleasure of having known NCL 🙃 . 

Python
 - Visualization 
    - Plot Types
 - Analysis
    - Dates and Times (or maybe we have another view or categorization structure?  probably a later issue

NCL to Python
 - Functions, Function Index, or NCL Index (one of these)
 - Categories or By Category (or similar adjustments to above)

This might also be something that we have better ideas about in the future.  We can put a pin in it too.

Reply:
That's a really good point. Hmm. I know we may have already gone past this point, but I don't totally feel like we should make that level of a structural change in this PR.

How do you feel about leaving this part in this PR alone and opening a separate issue/PR to be merged after this round of open PRs get merged? Then, we can have a more obvious place to discuss this, too.

Reply:
Totally fine!

Reply:
(see #54)